test/common/tmpdir.js

```diff
@@ -83,7 +83,7 @@ function rmdirSync(p, originalEr) {
           rimrafSync(path.join(p, f));
         }
       });
-      fs.rmdirSync(p);
+      rmdirSync(p, null);
```

Comment:
Does there need to be some sort of retry limit to avoid infinite loops?

Reply:
I thought about it but it should only happen if someone is infinitely (or create-check-retry) creating files in a directory we are trying to remove.
Though, I can add a retry counter as a third argument just to be sure WDYT?

Reply:
Well there's got to be a reason that [`rimraf`](https://www.npmjs.com/package/rimraf) by default caps the number of retries, so I think some sort of limit would be good.

Long term it would be good if https://github.com/nodejs/node/pull/30074 (which removes the custom `rimraf` implementation) could be fixed up.

Reply:
Updated.